### PR TITLE
Add openshift/cloud-provider-ibm repository

### PR DESCRIPTION
**Pull request by jonesbr17**
CI integration for https://github.com/openshift/cloud-provider-ibm

### CODE REVIEW

File: ci-operator/config/openshift/cloud-provider-ibm/openshift-cloud-provider-ibm-master.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+binary_build_commands: make commands
```
Comment (rtheis) [inline ci-operator/config/openshift/cloud-provider-ibm/openshift-cloud-provider-ibm-master.yaml:1]: Why aren't we running our full Travis bucket of linting and image building?

File: ci-operator/config/openshift/cloud-provider-ibm/openshift-cloud-provider-ibm-master.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.9"
+  namespace: ocp
+tests:
+- as: e2e-aws
```
Comment (rtheis) [inline ci-operator/config/openshift/cloud-provider-ibm/openshift-cloud-provider-ibm-master.yaml:22]: I assume all this AWS stuff is required until we have our own E2E?  If so, can we add a TODO comment to fix this?

File: ci-operator/jobs/openshift/cloud-provider-ibm/openshift-cloud-provider-ibm-master-presubmits.yaml [hunk 3]
```diff
@@ -0,0 +1,167 @@
+presubmits:
+  openshift/cloud-provider-ibm:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    cluster: build01
+    context: ci/prow/e2e-aws
```
Comment (rtheis) [inline ci-operator/jobs/openshift/cloud-provider-ibm/openshift-cloud-provider-ibm-master-presubmits.yaml:8]: Similar comment about AWS in this part.

**Review by rtheis — CHANGES_REQUESTED** [review]

**Comment by Fedosin** [comment]
I believe this one is a duplicate of https://github.com/openshift/release/pull/20427

**Comment by jonesbr17** [comment]
Closing in favor of https://github.com/openshift/release/pull/20427

**Pull request closed without merging**